include/votca/tools/elements.h

```diff
@@ -0,0 +1,655 @@
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef __VOTCA_TOOLS_ELEMENTS_H
+#define	__VOTCA_TOOLS_ELEMENTS_H
+
+#include <string>
+#include <map>
+#include <stdexcept>
+#include <boost/numeric/ublas/matrix.hpp>
+#include <boost/numeric/ublas/vector.hpp>
+#include <boost/numeric/ublas/io.hpp>
+#include <iostream>
+#include <fstream>
+
+namespace votca { namespace tools {
+
+namespace ub = boost::numeric::ublas;
+using namespace std;
+
+/**
+    \brief information about an element
+ 
+    The Elements class stores properties of several elements in the periodic table includeing: mass, element name, element symbol, van der Waals radius, effective nuclear potential, element number as it appears in the periodic table.  
+    
+*/
+class Elements
+{
+public:   
+
+    Elements() { FillMaps(); };
+   ~Elements() { };
+
+    /// ChelpG is a method for calculating the electrostatic potential outside 
+    /// of a molecule. The VdWChelpG radius in this case is specific to each 
+    /// element and defines the radius outside of which the electrostatic 
+    /// potential is calculated. The electrostatic potential within the radius
+    /// is not calculated. For more information see the reference paper CHELPG
+    /// paper [Journal of Computational Chemistry 11, 361, 1990]
+    const double     &getVdWChelpG(std::string name ) const 
+    { 
+        if(_VdWChelpG.count(name)==0) throw invalid_argument("Element not found in VdWChelpG map " + name);
+        return _VdWChelpG.at(name); 
+    }
+    
+    /// Merz-Singh-Kollman MK method is a similar method for calculating the 
+    /// electrostatic potential outside of a molecule. Details of the method can
+    /// be found here [Singh, U. Chandra (04/01/1984). "An approach to computing
+    /// electrostatic charges for molecules". Journal of computational chemistry
+    /// 5 (2), 129, 1984]. The VdWMK method will return the radii used to define
+    /// where the electrostatic grid starts.  
+    const double     &getVdWMK(std::string name )     const 
+    { 
+        if(_VdWMK.count(name)==0) throw invalid_argument("Elmenent not found in VdWMP map " + name);
+        return _VdWMK.at(name);     
+    }
+
+    /// This method is now deprecated but was originally used for determining 
+    /// the effective nuclear potential of an atom. There are now more 
+    /// appropriate methods for doing this in the basisset.h and aobasis.h file
+    /// in xtp. 
+    const double     &getNucCrgECP(std::string name)  const 
+    { 
+        throw invalid_argument("CrgECP map is deprecated");
+        return _NucCrgECP.at(name); 
```

Comment:
Remove return statement, it will never happen.

Reply:
Yes, but won't that trigger warning when travis runs?

Reply:
I think that there shouldn't be any warning as there is a `throw` statement.